app/models/auth/oidc.rb

```diff
@@ -23,7 +23,7 @@ def unified_credentials(hash)
       private
 
       def check_hash!(hash)
-        raise Errors::AuthError, 'invalid oidc credential hash' \
+        raise Errors::AuthenticationError, 'invalid oidc credential hash' \
           unless ENV_NAMES.values.all? { |key| hash.key?(key) }
```

Comment:
I would say it's less readable. I let @dudoslav decide. I leave it as it is for now.